lib/rubocop/cop/solidus/spree_t_decprecated.rb

```diff
@@ -54,6 +54,7 @@ def on_send(node)
           end
         end
 
+        # rubocop:disable Metrics/MethodLength
```

Comment:
The name of this cop is `Decprecated`. Is that right? 🤔

Reply:
Typo fixed in 90d4f88fa6b029ef4b2fd97bbed6503174c03cf7